### ISSUE TITLE
When going through the steps, my command executor gives these messages.

### ISSUE THREAD

**Issue by tytyG** [aboood40091/Miyamoto#85]
The term 'py' is not recognized as the name of a cmdlet, function, script file, or opera
ble program. Check the spelling of the name, or if a path was included, verify that the 
path is correct and try again.
The term 'python3' is not recognized as the name of a cmdlet, function, script file, or opera
ble program. Check the spelling of the name, or if a path was included, verify that the 
path is correct and try again.
At the steps where I need to use a command executor, I try to run the listed commands and type what is listed but then receive these error messages. I already have all of the files and have extracted all of the zip files.

**Comment by aboood40091** [comment]
Install Python and add it to path.
tl;dr read the readme.
